packages/@react-aria/aria-modal-polyfill/docs/aria-modal-polyfill.mdx

```diff
@@ -0,0 +1,50 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/aria-modal-polyfill';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import packageData from '@react-aria/aria-modal-polyfill/package.json';
+
+---
+category: Overlays
+keywords: [dialog, modal, aria]
+---
+
+# aria-modal-polyfill
```

Comment:
Maybe make the title `watchModals`? All the other docs pages have the title be the function/component name but this one has the package name. If you change it, make sure to change the filename of this mdx file too.